mods/cubyz/rotation/stairs.zig

```diff
@@ -276,8 +276,8 @@ fn closestRay(comptime typ: enum {bit, intersection}, block: Block, relativePlay
 pub fn rayIntersection(block: Block, item: ?main.items.Item, relativePlayerPos: Vec3f, playerDir: Vec3f) ?RayIntersectionResult {
 	if(item) |_item| {
 		switch(_item) {
-			.baseItem => |baseItem| {
-				if(std.mem.eql(u8, baseItem.id(), "cubyz:chisel")) { // Select only one eighth of a block
+			.tool => |tool| {
+				if(std.mem.eql(u8, tool.type.id(), "cubyz:chisel")) { // Select only one eighth of a block
```

Comment:
I still don't like that it's hardcoded, but I guess we can take one step at a time.